api/shapes/loadouts.ts

```diff
@@ -44,6 +44,14 @@ export interface Loadout {
   createdAt?: number;
   /** When was this Loadout last changed? Tracked automatically by the API - when saving a loadout this field is ignored. */
   lastUpdatedAt?: number;
+  /**
+   * Automatically added stat mods. These are separate from the manually chosen
+   * mods in parameters.mods, and are saved here to avoid having to figure them
+   * out all over again every time (especially since our algorithm might
+   * change). Combine this list and parameters.mods when displaying or actually
+   * applying the loadout.
+   */
+  autoStatMods?: number[];
```

Comment:
Would we be saving mods like powerful friends and subclass fragments in this when we get around to adding those?

Reply:
Only if we automatically added them. Otherwise they are chosen by the user and go in the usual place.